src/framework/audio/internal/audioconfiguration.cpp

```diff
@@ -20,17 +20,14 @@
  * along with this program.  If not, see <https://www.gnu.org/licenses/>.
  */
 #include "audioconfiguration.h"
-#include "settings.h"
-#include "stringutils.h"
-
-#include "global/deprecated/xmlreader.h"
-#include "global/deprecated/xmlwriter.h"
-
-#include "log.h"
 
 //TODO: remove with global clearing of Q_OS_*** defines
 #include <QtGlobal>
```

Comment:
Do we still need it?

Reply:
Yes, Q_OS_WASM is still used in this file